### ISSUE TITLE
Cannot center dataColumn label

### ISSUE THREAD

**Issue by deadsoul44** [flutter/flutter#40526]
I cannot center align dataColumn label as seen in the following picture although I wrap the label in center widget.

![image](https://user-images.githubusercontent.com/31016182/64923421-18c64480-d7da-11e9-864f-118f02c14147.png)

When I comment the following code section in data_table.dart file, everything works as expected as in the following picture. 

    /*
    if (onSort != null) {
      final Widget arrow = _SortArrow(
        visible: sorted,
        down: sorted ? ascending : null,
        duration: _sortArrowAnimationDuration,
      );
      const Widget arrowPadding = SizedBox(width: _sortArrowPadding);
      label = Row(
        textDirection: numeric ? TextDirection.rtl : null,
        children: <Widget>[ label, arrowPadding, arrow ],
      );
    }
    */

![image](https://user-images.githubusercontent.com/31016182/64923436-398e9a00-d7da-11e9-98ec-ae8ef79b5c2f.png)

I think arrow widget still takes some space although onSort is null. 

See the issue also on Stack Overflow:

https://stackoverflow.com/questions/56615116/how-do-you-center-the-label-in-a-flutter-datacolumn-widget/57945257#57945257

My Flutter Doctor:
```
[√] Flutter (Channel stable, v1.9.1+hotfix.2, on Microsoft Windows [Version 10.0.17134.1006], locale en-US)

[√] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
[√] Android Studio (version 3.5)
[√] Connected device (1 available)

• No issues found!
```

**Comment by deadsoul44** [comment]
It has nothing to do with stackoverflow. There is a bug in data_table.dart.

**Comment by j-kutz** [comment]
Is there any update on this?

**Comment by deadsoul44** [comment]
It is fixed with #51667. It is currently on dev channel.
